docs/apps/blender.md

```diff
@@ -0,0 +1,63 @@
+---
+tags:
+  - Free
+---
+
+# Blender
+
+Blender is the free, open source 3D creation suite which supports a variety of graphical applications such as 3D pipeline—modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation. 
+Advanced users can also use Blender’s API for Python scripting to customize the application and write specialized tools. 
```

Comment:
I am not sure if this is supported, so maybe best to remove the sentence about the Python API?

Reply:
Done